src/pudl/helpers.py

```diff
@@ -2281,22 +2280,22 @@ def get_parquet_table(
 
     try:  # Attempt to read as a GeoDataFrame
         df = gpd.read_parquet(
-            parquet_path,
+            path=parquet_path,
             columns=columns,
             filters=filters,
             schema=pyarrow_schema,
             use_threads=True,
             memory_map=True,
         )
     except (ValueError, TypeError):  # Fall back to normal Pandas DataFrame
-        df = pq.read_table(
-            source=parquet_path,
-            schema=pyarrow_schema,
+        df = pd.read_parquet(
```

Comment:
non-blocking: curious why you decided to split the IO managers into geo/normal but not this `get_parquet_table` function? This is fine, it works, etc. just wondering about your intent.